nifi-toolkit/nifi-toolkit-cli/src/main/java/org/apache/nifi/toolkit/cli/impl/command/CommandOption.java

```diff
@@ -164,7 +164,9 @@ public enum CommandOption {
     OUTPUT_TYPE("ot", "outputType", "The type of output to produce (json or simple)", true),
     VERBOSE("verbose", "verbose", "Indicates that verbose output should be provided", false),
     RECURSIVE("r", "recursive", "Indicates the command should perform the action recursively", false),
-    HELP("h", "help", "Help", false)
+    HELP("h", "help", "Help", false),
+    SKIP("skip", "skip", "Indicates to skip an operation if target object exists", true),
+    KEEP("keep", "keep", "Indicates to keep the original format or type of an object", true)
```

Comment:
Similar to `skip`, the word `keep` by itself does not indicate the meaning. Perhaps `preserve-format` or `preserve-type` could be better.